docs/troubleshooting.asciidoc

```diff
@@ -0,0 +1,29 @@
+[[troubleshooting]]
+== Troubleshooting
+
+The client provides several options for logging that can help when things go wrong. Check out the extensive documentation on <<logging>>.
+
+If you are having trouble sending a request to Elasticsearch with the client, we suggest enabling `tracing` on the client and testing the cUrl command that appears in your terminal:
+
+[source,rb]
+----------------------------
+client = Elasticsearch::Client.new(trace: true)
+client.info
+curl -X GET -H 'x-elastic-client-meta: es=8.9.0,rb=3.2.2,t=8.2.1,fd=2.7.4,nh=0.3.2, User-Agent: elastic-t
+ransport-ruby/8.2.1 (RUBY_VERSION: 3.2.2; linux x86_64; Faraday v2.7.4), Content-Type: application/json' 'http://localhost:9200//?pretty'
+----------------------------
+
+Testing the curl command can help find out if there's a connection issue or if the issue is in the client code.
```

Comment:
If you have accepted my previous suggestion, then please accept this, too, for consistency.

```suggestion
Testing the cURL command can help find out if there's a connection issue or if the issue is in the client code.
```